src/components/RuleCard/index.jsx

```diff
@@ -784,9 +784,8 @@ function RuleCard({
             <Link
               className={classes.link}
               to={{
-                pathname: '/rules/create',
+                pathname: `/rules/${rule.rule_id}/duplicate`,
                 state: {
-                  rule,
                   rulesFilter,
                 },
```

Comment:
![Screen Shot 2019-10-10 at 2 11 15 PM](https://user-images.githubusercontent.com/3766511/66594878-eead3b80-eb67-11e9-9cb1-0a88c4c33973.png)

Hmm, clicking "Duplicate" for ^ takes the user to http://localhost:9000/rules/undefined/duplicate. Is there a workaround that comes to your mind to fix this issue?

Reply:
Hm, I didn't realize we had the duplicate button for scheduled additions! The most obvious way to fix this to me is to have two new routes: /rules/duplicate/ruleId/:ruleId and /rules/duplicate/scId/:scId (or something like that). I'll see if I can come up with a better way, though.

Reply:
This turns out to be a pretty good fix, I think, since we already handle ruleId/scId in the params.